src/models/layerMetadata/layerMetadata.ts

```diff
@@ -14,17 +14,18 @@ import { getPyCSWMapping, IPYCSWMapping, pycsw } from './decorators/property/csw
 import { getInputDataMapping, IDataMapping, DataFileType, inputDataMapping } from './decorators/property/shp.decorator';
 import { getCatalogDBMapping, ICatalogDBMapping, catalogDB } from './decorators/property/catalogDB.decorator';
 import { getTsTypesMapping, ITsTypesMapping, tsTypes, TsTypes } from './decorators/property/tsTypes.decorator';
-import { ProductType, SensorType } from './enums';
+import { ProductType } from './enums';
 
 export interface ILayerMetadata {
```

Comment:
sensorType: SensorType[] | undefined;
You take it out from common, you need to add it to interface(I think)

Reply:
will fix it, i thought i did added it